packages/spoc_ppx/spoc_ppx.20210823/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "PPX to declare external GPGPU kernels written in CUDA or OpenCL"
+maintainer: ["mathias.bourgoin@gmail.com"]
+authors: ["Mathias Bourgoin"]
+license: "Cecill-B"
+homepage: "https://github.com/mathiasbourgoin/SPOC"
+bug-reports: "https://github.com/mathiasbourgoin/SPOC/issues"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "re"
+  "dune" {>= "2.9"}
+  "cppo"
+  "camlp4"
+  "ctypes"
+  "ctypes-foreign"
+  "ppxlib" {>= "0.22.0"}
+  "spoc"
```

Comment:
```suggestion
  "dune" {>= "2.9"}
  "ppxlib" {>= "0.22.0"}
  "spoc" {= version}
```
It looks also like this package only requires `spoc`, `ppxlib`, `dune` and `ocaml`